docs/wearables/sdks/authentication.mdx

```diff
@@ -0,0 +1,142 @@
+    Your app makes a `getVitalSignInToken` API call to your backend service using your own API authentication method.
+  </Step>
+  <Step title="Your backend -> Vital API">
+    Your backend service receives and validates the request, and forwards the request to the Vital [Create Sign-In Token API](/api-reference/user/create-sign-in-token) using your server-side Vital API Key secret.
+
+    ```bash
+    curl --request POST
+      --url '{{BASE_URL}}/v2/user/{{USER_ID}}/sign_in_token'
+      --header 'X-Vital-API-Key: <YOUR-API-KEY>'
+    ```
+  </Step>
+  <Step title="Vital API -> Your backend">
+    Vital API creates a new Vital Sign-In Token, and returns it to your backend service. Your backend service includes the Vital Sign-In Token as part of the `getVitalSignInToken` API response.
+  </Step>
+  <Step title="Your backend -> Your app">
+    Your app receives the API response and the Vital Sign-In Token. It then signs in with the Vital Mobile SDK using said token.
+
+    <CodeGroup>
+      ```swift Native iOS
+      import VitalCore
+
+      let response: MyAPIResponse
+
+      do {
+        try await VitalClient.signIn(withToken: response.signInToken)
+
+        print("Signed in with Vital successfully")
+      } catch let error {
+        print("Error signing in with Vital", error)
+      }
+      ```
+
+      ```kotlin Native Android
+      import io.tryvital.client.VitalClient
+      import android.content.Context
+      import android.util.Log
+
+      val response: MyAPIResponse
+      val applicationContext: Context
+
+      try {
+        VitalClient.signIn(applicationContext, response.signInToken)
+
+        Log.i("vital_sdk", "Signed in with Vital successfully")
+      } catch (e: Throwable) {
+        Log.e("vital_sdk", "Error signing in with Vital", e)
+      }
+      ```
+
+      ```typescript React Native
+      import { VitalCore } from "@tryvital/vital-core-react-native";
+
+      const response: MyAPIResponse
+
+      do {
+        await VitalCore.signIn(response.signInToken)
+
+        console.log("Signed in with Vital successfully")
+      } catch let error {
+        console.error("Error signing in with Vital", error)
+      }
+      ```
+
+      ```dart Flutter
+      import 'package:vital_core/vital_core.dart' as vital_core;
+
+      MyAPIResponse response;
+
+      try {
+        await vital_core.signIn(response.signInToken)
+
+        Fimble.i("Signed in with Vital successfully")
+      } catch (error) {
+        Fimble.e("Error signing in with Vital", error)
+      }
+      ```
+    </CodeGroup>
+
+  </Step>
+  <Step title="Voila!">
+    The Vital Mobile SDK is good to go!
+  </Step>
+</Steps>
+
+
+## Vital API Keys
+
+<Warning>
+API Key is **strongly discouraged** for production mobile apps, since your API Key would be
+effectively publicly distributed in cleartext. API Key support is intended only for customer
+early evaluation in Sandbox.
```

Comment:
> API Key is *discouraged* for production mobile apps, since it is distributed in cleartext. API Key support is intended only for customer early evaluation in Sandbox.